src/configuration/app-containers.md

```diff
@@ -467,17 +468,17 @@ hooks:
 
 ### Crons / Cron jobs
 
-Cron jobs allow you to run scheduled tasks at specified times or intervals. The `crons` object describes these tasks and the schedule that they are triggered on.  Each item in the list is a unique name identifying a separate cron job.
+Cron jobs allow you to run scheduled tasks at specified times or intervals. The `crons` object describes these tasks and the schedule when they are triggered.  Each item in the list is a unique name identifying a separate cron job.
 
-It has a few subkeys which are:
+It has a few subkeys listed below:
 
 -   **spec**: The [cron specification](https://en.wikipedia.org/wiki/Cron#CRON_expression). For example: `*/20 * * * *` to run every 20 minutes.
 -   **cmd**: The command that is executed, for example
     `cd public ; drush core-cron`
 
-The minimum interval between cron runs is 5 minutes, even if specified as less.
+The minimum interval between cron runs is 5 minutes; even if specified as less.
```

Comment:
"and" can join both independent (full sentence on its own) and dependent (incomplete sentence) clauses.  Semicolon only joins full sentences.  Since "even if specified as less" is not a complete sentence on is own a comma is more appropriate here.